File: AddressBook/AddressBook/Dtos/WebsiteLabel.cs
using AddressBook.Common.Paging;
using System.ComponentModel.DataAnnotations;

namespace AddressBook.Dtos
{
    public class WebsiteLabelRes
    {
        public int WebsiteLabelId { get; set; }
        public string? Label { get; set; }
    }

    public class WebsiteLabelReqEdit
    {
        [Required, MaxLength(20)]
        public string? Label { get; set; }
    }

    public class WebsiteLabelReqSearch : PagedReq
    {

    }
}

File: AddressBook/react-client/src/dtos/WebsiteLabel.ts
import Common from "../utility/Common";
import { PagedReq } from "./PagedReq";

export interface WebsiteLabelRes {
  websiteLabelId?: number;
  label?: string;
}

export class WebsiteLabelReqEdit {
  label?: string = "";
}

export class WebsiteLabelReqSearch extends PagedReq {
  constructor(
    {
      pageNumber = 1,
      pageSize = Common.DEFAULT_PAGE_SIZE,
      orderBy,
      searchText = "",
    }: PagedReq,
    {}
  ) {
    super({
      pageNumber: pageNumber,
      pageSize: pageSize,
      orderBy: orderBy,
      searchText: searchText,
    });
  }
}